### ISSUE TITLE
Lost some entities after HA Core 2023.11.0

### ISSUE THREAD

**Issue by wgumaa** [elupus/hass_nibe#194]
Hello all,

I had a good working integration up until last night when i updated to HA Core 2023.11.0. 

Some entities became unavailable for my S1255 heat pump and quite few other changes. 

I  noticed now that the sensor.priority_31029 doesn't have numerical values anymore but actual states; previously heating was  30 and hot water was 20; now they are HEAT for heating and HOT WATER etc. 

But what I lost were the following and not sure what has replaced them: 

1. sensor.compressor_status_31101 - Showed '0' - OFF and '1' - ON

2. number.hot_water_demand_mode_40057 - I was able to select the following '0.0': 'Small',  '1.0': 'Medium', '2.0': 'Large', '4.0': 'Smart Control'

If anyone can help and let me know what happened to these sensors or what they have been replaced with I would appreciate it.

Thanks in advance.

**Comment by elupus** [comment]
No s series pumps are supported by this integration. You must be using the built in version in home assistant. And no changes has been made for that parameter 31101 in core home assistant.

**Comment by wgumaa** [comment]
@elupus I am using the integration thats from the HA Integration page, but that's where things changed and decided to enquire. 

On my simple set up the values that were mapped for sensor.priority_31029 have changed and so I was able to fix that. 

However the 2 sensors I listed are unavailable and I get the "This entity is no longer being provided by the nibe_heatpump integration. If the entity is no longer in use, delete it in settings." message. So trying to understand what's going on. 

Would it be better to get rid of the HA integration and use this one?
